doc/manpages/siad.1

```diff
@@ -0,0 +1,156 @@
+.TH "SIAD" "1" "May 2017" "Auto generated by spf13/cobra" "siad Manual" 
+.nh
+.ad l
+
+
+.SH NAME
+.PP
+\&siad \- Sia Daemon 
+
+
+.SH SYNOPSIS
+.PP
+siad version
+.br
+siad modules
+.br
+siad [OPTIONS]
+
+.SH DESCRIPTION
+.PP
+The Sia daemon, siad, is a background process that does all of the heavy lifting for Sia. The daemon downloads the blockchain, uploads files, manages redundancy, monitors hosts for uptime, to name a few of the many tasks that it performs.
+The daemon is composed of modules, and each modules provides a set of primary functions on the network.
```

Comment:
This section has to be written manually, right? That's why I figured we would want the siad manpages to be static instead of autogenerated, because the autogenerated version wouldn't have a custom description. But if cobra can autogenerate manpages that contain a custom description, we can probably stick with that.